From c6a4047e20e95dd01e0c5ec629b77a50b3bc73d0 Mon Sep 17 00:00:00 2001
From: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
Date: Sun, 29 May 2016 23:18:49 +0300
Subject: [PATCH 004/122] spi: spi-gpio: set CPOL default inverted

Workaround:
 Set default value at probe to 1 due to issue with H3ULCB.HAD FPGA
 ethernet switch

Signed-off-by: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
---
 drivers/spi/spi-gpio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/spi/spi-gpio.c b/drivers/spi/spi-gpio.c
index 6e60521..428417d 100644
--- a/drivers/spi/spi-gpio.c
+++ b/drivers/spi/spi-gpio.c
@@ -289,7 +289,7 @@ static int spi_gpio_alloc(unsigned pin, const char *label, bool is_in)
 		if (is_in)
 			value = gpio_direction_input(pin);
 		else
-			value = gpio_direction_output(pin, 0);
+			value = gpio_direction_output(pin, 1);
 	}
 	return value;
 }
-- 
2.7.4

